[2024-05-15T07:17:25.266+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-15T07:17:25.344+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_and_parse_cris_data.fetch_cris_data scheduled__2024-05-08T06:00:00+00:00 [queued]>
[2024-05-15T07:17:25.378+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_and_parse_cris_data.fetch_cris_data scheduled__2024-05-08T06:00:00+00:00 [queued]>
[2024-05-15T07:17:25.380+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2024-05-15T07:17:25.429+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): fetch_cris_data> on 2024-05-08 06:00:00+00:00
[2024-05-15T07:17:25.449+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=57) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-15T07:17:25.462+0000] {standard_task_runner.py:63} INFO - Started process 101 to run task
[2024-05-15T07:17:25.457+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'fetch_and_parse_cris_data', 'fetch_cris_data', 'scheduled__2024-05-08T06:00:00+00:00', '--job-id', '1927', '--raw', '--subdir', 'DAGS_FOLDER/cris.py', '--cfg-path', '/tmp/tmp224xbfuu']
[2024-05-15T07:17:25.469+0000] {standard_task_runner.py:91} INFO - Job 1927: Subtask fetch_cris_data
[2024-05-15T07:17:25.602+0000] {task_command.py:426} INFO - Running <TaskInstance: fetch_and_parse_cris_data.fetch_cris_data scheduled__2024-05-08T06:00:00+00:00 [running]> on host e8691f41e037
[2024-05-15T07:17:25.801+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fetch_and_parse_cris_data' AIRFLOW_CTX_TASK_ID='fetch_cris_data' AIRFLOW_CTX_EXECUTION_DATE='2024-05-08T06:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-08T06:00:00+00:00'
[2024-05-15T07:17:25.805+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-15T07:17:47.044+0000] {logging_mixin.py:188} INFO - Failed to fetch data from the URL: 403
[2024-05-15T07:17:47.045+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-15T07:17:47.046+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-15T07:17:47.058+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=fetch_and_parse_cris_data, task_id=fetch_cris_data, execution_date=20240508T060000, start_date=20240515T071725, end_date=20240515T071747
[2024-05-15T07:17:47.156+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-15T07:17:47.200+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-15T07:17:47.206+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
